src/views/Home.jsx

```diff
@@ -1,16 +1,24 @@
 import './Home.css';
+import { SingleList } from '../components';
 
 export function Home({ data, setListPath }) {
+	const dataPresent = data.length !== 0;
```

Comment:
One option is to inline this check and inverting it

So use: (!Array.isArray(data) || data.length == 0)

Reply:
if we're worried about the array being null, you can use the "nullish coalescing operator" to give the prop a default value

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing
```
let list = null;

// assign a default at the top of our component
let listProp = list ?? [];
console.log(listProp) // logs "[]"
```

Reply:
I've seen the nullish coalescing operator before but didn't have a full grasp on how. Thanks for the docs and an example it makes so much more sense!